src/LogRequest.php

```diff
@@ -39,7 +39,7 @@ protected function getData($request) : array
             'ip' => $request->getClientIp(),
             'path' => urldecode($request->path()),
             'method' => $request->getMethod(),
-            'agent' => $request->server('HTTP_USER_AGENT'),
+            'agent' => substr($request->server('HTTP_USER_AGENT'), 0, 191),
```

Comment:
What if the varchar is more/less 191? Wouldn't a better solution be to just modify this column to text?

Reply:
But what if 65,535 is not enough space.. Actually I preferred that, too. But why would varchar be less than 191 chars? I mean at least for our use case that will never happen. 
PS: I discovered that comment yesterday, when i took a look at my spam folder, sorry for being late ;)

Reply:
If it would never happen, we would not need to make it a fixed length of 191.

> But what if 65,535 is not enough space.

What do you mean with that?

Reply:
we fixed the length because many mobile agents seem to have more than 191 chars in length, varchar did not change.

> If it would never happen, we would not need to make it a fixed length of 191.
> 
> > But what if 65,535 is not enough space.
> 
> What do you mean with that?

that is the length of the TEXT datatype in chars, at least for mysql.

Reply:
Then use text ... my humble assumption was that > 191 Chars is just plain trash / part of an attack which does not need to be logged. Looks like i was wrong

Reply:
> that is the length of the TEXT datatype in chars, at least for mysql.

TIL :upside_down_face: 

Though I don't think a user agent will ever outgrow that.